node/src/components/small_network/error.rs

```diff
@@ -21,15 +21,12 @@ pub enum Error {
     #[error("could not resolve at least one known host (or none provided)")]
     EmptyKnownHosts,
     /// Consensus signing during handshake was provided, but keys could not be loaded.
-    #[error("consensus keys provided, but could not be loaded")]
+    #[error("consensus keys provided, but could not be loaded: {0}")]
```

Comment:
All errors should be (in logs) display with `display_error`, which is at least the case inside of `small_network`. The function shows the complete chain of error sources. Adding the `{0}` here is redundant, as far as I know it'd make that output look like "consensus keys provided, but could not be loaded: foo load error: foo load error: bar error".

Reply:
Isn't that what it looks like everywhere else?

Reply:
Yes, unfortunately. But I would really prefer to see it changed anywhere else than here :).

At the very least, let's keep the small_network component consistent internally?

Reply:
Let's leave this for now and figure out a solution where everyone can be on the same page.